llvm/include/llvm/CodeGen/TargetRegisterInfo.h

```diff
@@ -630,6 +630,12 @@ class TargetRegisterInfo : public MCRegisterInfo {
     return false;
   }
 
+  /// Returns true if TargetRegisterClass is a general purpose register.
```

Comment:
a class/subclass of general purpose register.

Reply:
[449d640](https://github.com/llvm/llvm-project/pull/98603/commits/449d640bc8357dbb5b0bca4014019e5502570069)